cli/config/standard-libs.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Any idea if we can get dependabot to look at this file as dependencies?

Reply:
Probably if we made it a package.json, but only the names are actually used at present.  I tried to consolidate the standard deps into a common package so we didn’t commingle them or duplicate them (4 places currently).  I couldn’t get it to work nicely though, so left them as duplicates for now

Reply:
Alright, problems for a bright future to solve.